### PR TITLE
Export toast

### PR DESCRIPTION
**Pull request by seagyn**
Forgot to export the toast...

### CODE REVIEW

**Comment by seagyn** [comment]
@fjlaubscher @MrOpperman this has updates with the toast fix.

File: src/toast/style.js [hunk 1]
```diff
@@ -13,6 +13,122 @@ const Icon = css`
...
+  }
+
+  to {
+    opacity: 0;
+  }
+`;
+
+const slideInTop = keyframes`
+  0% {
+    -webkit-transform: translateY(-500px);
```
Comment (fjlaubscher) [inline src/toast/style.js]: You don't have to worry about browser prefixes, styled-components adds them automatically.

Comment (seagyn) [reply src/toast/style.js]: Ah thanks! Was wondering if keyframes left them in or not

**Pull request merged**